src/data/waffleFlags.js

```diff
@@ -0,0 +1,55 @@
+import { getConfig } from '@edx/frontend-platform';
```

Comment:
Can we add a link to [Feature Toggle HOWTO](https://github.com/openedx/frontend-app-payment/blob/master/docs/how_tos/feature_toggle.rst) somewhere appropriate in this file?